test/y2security/security_policies/manager_test.rb

```diff
@@ -91,6 +91,11 @@
   end
 
   describe "#enable_policy" do
+    before do
+      allow(subject).to receive(:require).with("installation/services")
```

Comment:
I don't get this. The require is mocked but the class `Installation::Services` is used below. Does it actually work if the require is mocked?

Reply:
Yes, because we are kind of mocking that class (see https://github.com/yast/yast-security/pull/134/files#diff-ba37813ca277c227a74a372479b7b05b7f3ff085d890ab708f80d62573efdb7a).